diff --git a/base/BUILD.gn b/base/BUILD.gn
index cf178fb32ef8112cb5d6ac1f094e5f3ce027ac71..51a3082ece7a754c0a63b345e9f8faa8c6030af6 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4473,6 +4473,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4512,6 +4513,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
