diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index 5d2d30c33ed0f8c13463d4c60c372d1168d0f968..b9d96e0c0861b153f2e967a7a855348494d09874 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -585,6 +585,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return nullptr;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -931,6 +932,7 @@ bool IsHostAllowedInIncognito(const GURL& url) {
   base::StringPiece host = url.host_piece();
   if (scheme != content::kChromeUIScheme)
     return true;
+  BRAVE_IS_HOST_ALLOWED_IN_INCOGNITO
 
   if (host == chrome::kChromeUIChromeSigninHost) {
 #if BUILDFLAG(IS_WIN)
