diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 8a20eb68b01fc0b059e0a705100d6c356108bf06..e32d8adecfe21a349eb7dfa359b7dc0c7bf0bedb 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -117,6 +117,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
