diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index ec8cdddfc75988966fda05a8409c2ade92296d07..216dac61b2a05cff66898edcd96964abbf8e54e2 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3869,6 +3869,7 @@ const FeatureEntry::FeatureVariation kAndroidHubVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
