diff --git a/components/password_manager/core/browser/login_database.cc b/components/password_manager/core/browser/login_database.cc
index 9c0b43549c541ce9879c4d0a23050d9108d82f4a..d1e17b4be98d3864d53e9220e46b7b6b11b3d911 100644
--- a/components/password_manager/core/browser/login_database.cc
+++ b/components/password_manager/core/browser/login_database.cc
@@ -2285,6 +2285,7 @@ FormRetrievalResult LoginDatabase::StatementToForms(
     std::u16string plaintext_password;
     EncryptionResult result =
         DecryptPasswordFromStatement(*statement, &plaintext_password);
+    BRAVE_STATEMENT_TO_FORMS
     if (result == ENCRYPTION_RESULT_SERVICE_FAILURE) {
       has_service_failure = true;
       continue;
