diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index b4236e0f6c2d9eec2dae09677d0755306b180868..ef8c23f52cbe83cf8b5504ae1156e64192f2b5f8 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1244,6 +1244,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
