diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index eacb5ec6a3cb81b3ba534afcb7d85971bb260741..c1cdb8d870331dddffdcd6c15b8ebe3cc140fc69 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -183,6 +183,7 @@ bool CanTrackLastVisit(ContentSettingsType type) {
   // a valid "initial default" value.
   if (type == ContentSettingsType::PROTOCOL_HANDLERS)
     return false;
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   auto* info =
       content_settings::ContentSettingsRegistry::GetInstance()->Get(type);
