diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index bfce9c15502789cb2465ea64f2fe11a762f045f1..83451ac6d077b87a16a887584eaa341aaf0e7654 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -251,6 +251,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
