diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 43d8df1d167302457ad263035e12ad4467715c0c..105e13f111d3043333ee450e06e955d0f1700978 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -735,8 +735,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
