diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 262dd61a5912b8c568ab9c96d3f0922a41b0c513..6202a5bf108b08eb6bda0ff5d901bef3f1a91ee6 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -735,8 +735,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
