diff --git a/net/dns/host_resolver_manager.cc b/net/dns/host_resolver_manager.cc
index 31174967b526461362fde663d9e217c8f3179ffe..2a87d8fc1411ceead81c8f29500221cfb1ff9d2e 100644
--- a/net/dns/host_resolver_manager.cc
+++ b/net/dns/host_resolver_manager.cc
@@ -2797,6 +2797,7 @@ class HostResolverManager::Job : public PrioritizedDispatcher::Job,
       MaybeCacheResult(results, ttl, secure);
 
     RecordJobHistograms(results, task_type);
+    BRAVE_RECORD_DNS_COUNTS
 
     // Complete all of the requests that were attached to the job and
     // detach them.
